vendor/github.com/BurntSushi/toml/.gitignore

```diff
@@ -0,0 +1,5 @@
+TAGS
```

Comment:
we can run a script to clean these file, you can see hack/clean_vendor.sh in tidb